roles/lib_openshift/library/oc_adm_ca_server_cert.py

```diff
@@ -1345,6 +1351,17 @@ def get(self):
 
     def create(self):
         '''run openshift oc adm ca create-server-cert cmd'''
+
+        # Added this here as a safegaurd for stomping on the
+        # cert and key files if they exist
+        if self.config.config_options['backup']['value']:
+            if os.path.exists(self.config.config_options['key']['value']):
+                shutil.copy(self.config.config_options['key']['value'],
+                            "%s.orig" % self.config.config_options['key']['value'])
+            if os.path.exists(self.config.config_options['cert']['value']):
+                shutil.copy(self.config.config_options['cert']['value'],
+                            "%s.orig" % self.config.config_options['cert']['value'])
```

Comment:
Would date stamped backups be better here? Ideally, I would like to see us use [backup_local](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L2055) from module_utils, but that would require passing module down into CAServerCert/OpenShiftCLI.

Having the date stamped backups will ensure that we wouldn't clobber the backups on a subsequent run.